.gitattributes

```diff
@@ -30,6 +30,4 @@
 *.physicMaterial    merge=unityyamlmerge eol=lf
 *.asset             merge=unityyamlmerge eol=lf
 *.meta              merge=unityyamlmerge eol=lf
-*.controller        merge=unityyamlmerge eol=lf
-
-Assets/Plugins/Linux/libDynamicLibraryLoaderHelper.so filter=lfs diff=lfs merge=lfs binary
```

Comment:
Is this removal intentional?

Reply:
Sort of, this PR was supposed to be all fixes post 3.2.0, there was only one thus the name.
I'll rename the PR

Reply:
Both those removes seems suspicious to me.

Reply:
Only the linux line is actually removed, the other one is because of line ending changes

Reply:
The linux line is moved to the .gitattribute under `Linux/`
Since there is no going back from lfs I suppose